src/multiplicity_vectors.lean

```diff
@@ -36,15 +36,16 @@ noncomputable theory
 
 section basics
 
+/--Two pnats are coprime if their gcd is 1-/
```

Comment:
```suggestion
/-- Two pnats are coprime if their gcd is 1. -/
```
Please add spaces between the comment tags and the text.

Reply:
I think I found all of them.